tests/CppUTestExt/IEEE754PluginTest_c.c

```diff
@@ -71,3 +75,5 @@ void set_everything_c()
     set_invalid_c();
     set_inexact_c();
 }
+
+#endif // CPPUTEST_HAVE_FENV
```

Comment:
Just remove the comment. That will make the compiler happy, and @basvodde, too ;-)